src/components/NewCommentForm.tsx

```diff
@@ -42,27 +101,37 @@ export const NewCommentForm: React.FC = () => {
         <div className="control has-icons-left has-icons-right">
           <input
             type="text"
-            name="email"
             id="comment-author-email"
             placeholder="email@test.com"
-            className="input is-danger"
+            className={cn('input', { 'is-danger': touchedEmail })}
+            value={email}
+            onChange={event => setEmail(event.target.value)}
+            onBlur={() => setTouchedEmail(true)}
           />
 
-          <span className="icon is-small is-left">
-            <i className="fas fa-envelope" />
-          </span>
-
-          <span
-            className="icon is-small is-right has-text-danger"
-            data-cy="ErrorIcon"
-          >
-            <i className="fas fa-exclamation-triangle" />
-          </span>
+          {
+            touchedEmail && (
+              <>
+                <span className="icon is-small is-left">
+                  <i className="fas fa-envelope" />
+                </span>
+                {!email && (
+                  <span
+                    className="icon is-small is-right has-text-danger"
+                    data-cy="ErrorIcon"
+                  >
+                    <i className="fas fa-exclamation-triangle" />
+                  </span>
+                )}
+                {!email && (
+                  <p className="help is-danger" data-cy="ErrorMessage">
+                    Email is required
+                  </p>
+                )}
```

Comment:
You can merge those elements with React.Fragment, because the conditions are same